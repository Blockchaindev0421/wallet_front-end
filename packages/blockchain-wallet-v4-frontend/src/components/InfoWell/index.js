import styled from 'styled-components'

const InfoWell = styled.div`
  padding: 15px;
  margin-bottom: 10px;
  box-sizing: border-box;
  background-color: ${props => props.theme['brand-yellow-lighter']};
  border-left: 5px solid ${props => props.theme['brand-yellow']};
  text-align: justify;

  & > * { display: inline; margin-right: 5px; }
`

export default InfoWell
