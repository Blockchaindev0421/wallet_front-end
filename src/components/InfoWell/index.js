import styled from 'styled-components'

const InfoWell = styled.div`
  padding: 15px;
  margin-bottom: 10px;
  box-sizing: border-box;
  background-color: #FFE6B4;
  border-left: 5px solid #FFCF62;
  text-align: justify;

  & > * { display: inline; margin-right: 5px; }
`

export default InfoWell
