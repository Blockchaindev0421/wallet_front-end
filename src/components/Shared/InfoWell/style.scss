
.info-well {
  margin: 5px 0px 10px 0px;
  padding: 15px;
  background-color: #FFE6B4;
  border-left: 5px solid #FFCF62;
  line-height: 1.5rem;
  font-size: 0.9rem;
  font-weight: 400;
}
