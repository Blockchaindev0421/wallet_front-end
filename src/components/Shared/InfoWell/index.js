import React from 'react'
import CSSModules from 'react-css-modules'

import style from './style.scss'

const InfoWell = ({ children }) => (
  <div styleName='info-well'>
    {children}
  </div>
)

export default CSSModules(InfoWell, style)
