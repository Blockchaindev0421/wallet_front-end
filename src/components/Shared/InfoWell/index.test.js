import React from 'react'
import renderer from 'react-test-renderer'

import InfoWell from './index.js'

test('Displays an info well', () => {
  const component = renderer.create(
    <InfoWell>Hello</InfoWell>
  )
  expect(component.toJSON()).toMatchSnapshot()
})
