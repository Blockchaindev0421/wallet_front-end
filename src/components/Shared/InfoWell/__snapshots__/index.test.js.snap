// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays an info well 1`] = `
<div
  className="info-well"
>
  Hello
</div>
`;
